lib/watir_drops/page_object.rb

```diff
@@ -18,52 +20,68 @@ def page_url
         end
       end
 
+      def page_title
+        define_method("page_title") do |args|
+          yield(*args)
+        end
+      end
+
       def element_list
         @element_list ||= []
       end
 
+      def required_element_list
+        @required_element_list ||= []
+      end
+
       def inherited(subclass)
         subclass.element_list = element_list.dup
+        subclass.required_element_list = required_element_list.dup
       end
 
-      def elements(name, &block)
+      def elements(name, required = false, &block)
```

Comment:
A test would verify this (hint hint), but I don't think adding a ElementCollection instance to this list would work because it doesn't respond to `#present?`. Plus I don't think we need it.